helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java

```diff
@@ -131,6 +131,7 @@
   private int _reportLatency;
 
   protected RealmAwareZkClient _zkclient;
+  private RealmAwareZkClient.RealmAwareZkConnectionConfig _realmAwareZkConnectionConfig;
```

Comment:
Can we make this field final too? Since it is an input config.

Reply:
It is not given via ZkHelixManager's constructor, meaning it could be null. It comes from HelixManagerProperty as an optional field. So we cannot make this final.